### PR TITLE
Some TODOs

### PR DESCRIPTION
**Pull request by noobadmin**
## TODOs resolved

- [ ] minor design adjustments (max page width, spaces around min/max/unit fields, smaller font size for filter fields, ExPaNDS logo, etc.)
- [x] review filter groups feature (simplify code if no grouping is needed for now)
- [ ] feature testing with React Testing Library
- [ ] let user adjust limit
- [ ] show institute in simple view
- [x] detailed view toggle:
  - [x] full title
  - [ ] datasets matching the filters (make it clear that there may be more)
- [x] expandable description in both simple and detailed views
- [ ] fancy scoring indicator
- [x] way to provide feedback
- [ ] techniques auto-complete drop-down
- [ ] release date filter
- [ ] suggested searches on homepage (text field placeholder + links under text field)
- [x] footer/privacy statement?
- [ ] track queries so we know what people search for? (in back-end?)
- [ ] debug query URL/params at the bottom

### CODE REVIEW

File: .eslintrc.cjs [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+const { files: jestFiles } = require('eslint-config-galex/dist/overrides/jest')
+const {
+  files: reactFiles,
+} = require('eslint-config-galex/dist/overrides/react')
 
 module.exports = createConfig({
+  env: {
+    es6: true,
+    browser: true,
+  },
```
Comment (axelboc) [inline .eslintrc.cjs:11]: this shouldn't be needed in theory 🤷

File: src/App/helpers.js [hunk 2]
```diff
@@ -1,13 +1,16 @@
-import moment from 'moment'
+import { format, isValid } from 'date-fns'
```
Comment (axelboc) [inline src/App/helpers.js:1]: Good call replacing `moment` 👍

File: src/App/helpers.js [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
   return `${str[0].toUpperCase()}${str.slice(1).replaceAll('_', ' ')}`
 }
 
-export function documentSize(datasets) {
-  return `${true} MB`
-}
+const isEmpty = (obj) => JSON.stringify(obj) === '{}'
+export const stripEmptyKeys = (obj) =>
+  Object.fromEntries(
+    Object.entries(obj).filter(([, v]) => v !== undefined && !isEmpty(v)),
```
Comment (axelboc) [inline src/App/helpers.js:15]: Is `v` guaranteed to be an object or `undefined`? If so, I think you can just check for `v && Object.keys(v).length > 0`

File: src/Explore/Detail.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+function Title({ expand, onClick, text }) {
+  return (
+    <Text onClick={onClick} fontWeight="bold">
+      {`${text} ${expand ? '\u2227' : '\u2228'}`}
+    </Text>
+  )
+}
+function Details(props) {
+  const { open, title } = props
+  const [expand, setExpand] = useState(open)
```
Comment (axelboc) [inline src/Explore/Detail.js:51]: I'd rename this to past tense - i.e. `[expanded, setExpanded]`

Actually you can use `useToggle`: https://react-hookz.github.io/web/?path=/docs/state-usetoggle--example

```ts
const [expanded, toggleExpanded] = useState(open);
onClick={() => toggleExpanded()}
```

File: src/Explore/DocumentItem.js [hunk 5]
```diff
@@ -1,13 +1,63 @@
...
+          url={`/documents/${encodeURIComponent(pid)}`}
+          mergeFn={getMembers}
+          rowId="id"
+          title="Members"
+          config={{ include: ['person', 'affiliation'] }}
+        />
+      </>
+    )
+  }
+  function Poor() {
```
Comment (axelboc) [inline src/Explore/DocumentItem.js:40]: Maybe `Simplified` and `Detailed`? 😁

**Review by axelboc — APPROVED** [review]

File: src/App/helpers.js (hunk 3 above)
Comment (noobadmin) [reply src/App/helpers.js:15]: Unfortunately not

**Pull request merged**